editor/src/components/text-editor/text-editor.tsx

```diff
@@ -362,11 +364,11 @@ async function setSelectionToOffset(
   }
 }
 
-function stopPropagation(e: React.KeyboardEvent | React.ClipboardEvent) {
+function stopPropagation(e: React.KeyboardEvent | React.ClipboardEvent | React.MouseEvent) {
```

Comment:
FYI you could use `React.UIEvent` here

Reply:
ClipboardEvent is not part of that, but still a good tip, I replace the KeyboardEvent and the MouseEvent with that.